### ISSUE TITLE
Allow CMake build without network access

### ISSUE THREAD

**Issue by PureTryOut** [libpinyin/libpinyin#121]
It seems the CMake configuration currently downloads the model data from SourceForge without anyway to stop that from happening and provide the data ourselves.

https://github.com/libpinyin/libpinyin/blob/master/data/CMakeLists.txt#L43

It would be great if we could pass an argument to CMake to stop that from happening, so build systems can work completely offline.

**Comment by epico** [comment]
Please extract the model17.text.tar.gz into ${CMAKE_SOURCE_DIR}/data/ , it will work I think.

Fixed in https://github.com/epico/libpinyin .

**Comment by PureTryOut** [comment]
Yeah I made #122 to stop wget from downloading it, and downloading and extracting the model to that directory works. I already did that yesterday :wink: 

This issue is here more as a reference.

**Comment by epico** [comment]
Sorry, I mean without #122 PR, just extract the files into ${CMAKE_SOURCE_DIR}/data, it will also avoid to download the model17.text.tar.gz file.

**Comment by felixonmars** [comment]
It works here with cmake & make, but not with cmake & ninja. In `cmake -GNinja` the wget commands are always executed.
